sonarqube/docker-healthcheck

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# A Sonarqube container is considered healthy if the status is UP, DB_MIGRATION_NEEDED or DB_MIGRATION_RUNNING
+# status about migration are added to prevent the node to be kill while sonarqube is updating himself.
+
+if curl -Ss http://localhost:9000/api/system/status | grep -q -e '"status":"UP"' -e '"status":"DB_MIGRATION_NEEDED"' -e '"status":"DB_MIGRATION_RUNNING"'; then
```

Comment:
Shouldn't this be attempting to connect to the external interface instead to better verify the container's purported external health?

(As in https://github.com/docker-library/healthcheck/blob/436d3437d3b75f41d3518720c077df3e8d3570ce/cassandra/docker-healthcheck#L4 andd other examples here?)

Reply:
Yes it is ! thanks, i'm updating the PR to do so.